faker.go

```diff
@@ -108,6 +108,7 @@ const (
 	AmountWithCurrencyTag = "amount_with_currency"
 	SKIP                  = "-"
 	Length                = "len"
+	SliceLength           = "slice_len"
```

Comment:
is it possible to use `len` instead? to make the same experience?

Reply:
Shouldn't it be possible to use both? 
Have a slice of []string where you can set both the length of the slice and of each string in the slice?

Reply:
Ahh, I see. That makes sense